app/src/main/res/xml/prefactions_share_into.xml

```diff
@@ -40,7 +40,6 @@
         android:title="@string/calendar" />
 
     <PreferenceCategory
-        android:key="@string/pref_key__favourite_files"
```

Comment:
Removed as android studio complained about this key being a duplicate. Don't think there is any detriment to removing it.

Reply:
I see, seems like copy paste error